internal/proto/consts.go

```diff
@@ -0,0 +1,16 @@
+package proto
+
+const (
+	// Version is the latest version of the protocol. It is implicitly 0 for
+	// clients that didn't yet have a protocol version
+	Version = 1
+	// V0NotifyReady is the value sent at the end in the v0 protocol to indicate
```

Comment:
If this wasn't so nice and clean, I wouldn't have said anything, but you're missing a blank line between the previous value and the next comment.